datacrunch/instances/instances.py

```diff
@@ -123,7 +123,12 @@ def create(self,
                is_spot: bool = False,
                contract: Optional[Contract] = None,
                pricing: Optional[Pricing] = None,
-               coupon: Optional[str] = None) -> Instance:
+               coupon: Optional[str] = None,
+               *,
```

Comment:
`*` is a keyword-only argument marker, probably a good idea to do it for all new arguments.

https://docs.python.org/3/glossary.html#term-parameter